unstructured/ingest/evaluate.py

```diff
@@ -82,17 +85,11 @@ def measure_text_edit_distance(
     Calculates text accuracy and percent missing. After looped through the whole list, write to tsv.
     Also calculates the aggregated accuracy and percent missing.
     """
-
-    print(f"output_dir is {output_dir}")
-    print(f"source_dir is {source_dir}")
     if not output_list:
         output_list = _listdir_recursive(output_dir)
     if not source_list:
         source_list = _listdir_recursive(source_dir)
 
-    print(f"output_list is {output_list}")
-    print(f"source_list is {source_list}")
-
     if not output_list:
```

Comment:
When I tested, I'm getting this empty output list actually:
```
shreyanid@Shreyas-MacBook-Pro-2 unstructured % sh ./test_unstructured_ingest/evaluation-metrics.sh text-extraction
download: s3://utic-dev-tech-fixtures/small-eval-text-extraction/science-exploration-1p.pptx.txt to test_unstructured_ingest/gold-standard/small-eval-text-extraction/science-exploration-1p.pptx.txt
download: s3://utic-dev-tech-fixtures/small-eval-text-extraction/IRS-form-1987.pdf.txt to test_unstructured_ingest/gold-standard/small-eval-text-extraction/IRS-form-1987.pdf.txt
download: s3://utic-dev-tech-fixtures/small-eval-text-extraction/handbook-1p.docx.txt to test_unstructured_ingest/gold-standard/small-eval-text-extraction/handbook-1p.docx.txt
download: s3://utic-dev-tech-fixtures/small-eval-text-extraction/example-10k.html.txt to test_unstructured_ingest/gold-standard/small-eval-text-extraction/example-10k.html.txt
No output files to calculate to edit distances for, exiting
```

Reply:
update: it's because I don't have the unstructured output files to compare in the `structured-output-eval` folder. waiting to receive those to add manually and will retest

Reply:
received the files, test worked for me now!